js/bitcoincoid.js

```diff
@@ -310,9 +310,17 @@ module.exports = class bitcoincoid extends Exchange {
     }
 
     async cancelOrder (id, symbol = undefined, params = {}) {
+        if (!symbol)
+            throw new ExchangeError (this.id + ' cancelOrder requires a symbol argument');
+        if ('side' in params)
+            if (!params['side'])
+                throw new ExchangeError (this.id + ' cancelOrder requires a side as params argument');
         await this.loadMarkets ();
+        let market = this.market (symbol);
         return await this.privatePostCancelOrder (this.extend ({
             'order_id': id,
+            'pair': market['id'],
+            'type': params['side']
```

Comment:
One single comma missing at the end of this line to satisfy the syntax checkers.

Reply:
Next PR 💃